app/views/planned_runs/index.html.erb

```diff
@@ -26,21 +23,21 @@
       <% else %>
           <div class="row" style="background-color: lightblue">
       <% end %>
-        <div class="col-md-3 col-sm-3 col-xs-3">
+        <div class="col-md-4 col-sm-3 hidden-xs index_rows">
           <%= planned_run.date %>
         </div>
-        <div class="col-md-3 col-sm-3 col-xs-2">
-          <%= Group.find(planned_run.group_id).name %>
+        <div class="hidden-lg hidden-md hidden-sm col-xs-3 index_rows">
+          <%= Date.strptime(planned_run.date.to_s, "%Y-%m-%d").strftime("%m-%d") %>
```

Comment:
Just used .strftime and it seemed to do the job: https://github.com/o19s/thats-trackable/commit/7c652bd8b733f75dc6a5aac434191e759925dea5